floating_ips_actions.go

```diff
@@ -9,6 +9,7 @@ type FloatingIPActionsService interface {
 	Assign(ip string, dropletID int) (*Action, *Response, error)
 	Unassign(ip string) (*Action, *Response, error)
 	Get(ip string, actionID int) (*Action, *Response, error)
+	List(ip string) ([]Action, *Response, error)
```

Comment:
nps :) I am just going by the `Get(ip string, actionID int) (*Action, *Response, error)` :)